WcaOnRails/config/i18n-tasks.yml.erb

```diff
@@ -67,7 +67,9 @@ search:
     - app/assets/images
     - app/assets/fonts
     # i18n-tasks fails on TNoodle jars and old non-utf8 translations...
-    - app/views/regulations
+    - app/views/regulations/history
```

Comment:
Yep, `app/views/regulations` was excluded from the translations. I added `app/views/regulations/about`, so I had to change this here to make the translations work for the about page.